config/config.go

```diff
@@ -30,7 +30,8 @@ type Config struct {
 	MaxNFReStartCount int
 	MaxNFsAttachCount int
 	Environment       string
-
+	BpfMapDefaultDir  string
+	TcMapsRelativeDir string
```

Comment:
"Dir" implies the path is a filesystem directory, which is not the case on Windows.  I'd recommend "Path" (or "BasePath" would also be ok) instead, throughout.
```suggestion
	BpfMapDefaultPath  string
	TcMapsRelativePath string
```
thus the "full path" = the "path" (or "base path") + "name"

Reply:
Should I change in l3afd cfg file or change in config go file is enough,
Because in cfg file there are other variables  having     "dir" suffix

Reply:
Dir is ok if something really is a file.  But pin paths on Windows are not files.